fenix/app/src/main/java/org/mozilla/fenix/compose/ThumbnailImage.kt

```diff
@@ -20,37 +21,36 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.asImageBitmap
 import androidx.compose.ui.graphics.painter.BitmapPainter
 import androidx.compose.ui.layout.ContentScale
-import androidx.compose.ui.platform.LocalContext
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
 import kotlinx.coroutines.launch
-import mozilla.components.browser.thumbnails.storage.ThumbnailStorage
 import mozilla.components.concept.base.images.ImageLoadRequest
+import org.mozilla.fenix.components.components
 import org.mozilla.fenix.theme.FirefoxTheme
 
 /**
  * Thumbnail belonging to a [ImageLoadRequest]. Asynchronously fetches the bitmap from storage.
  *
  * @param request [ImageLoadRequest] used to fetch the thumbnail bitmap.
- * @param storage [ThumbnailStorage] to obtain tab thumbnail bitmaps from.
- * @param modifier [Modifier] used to draw the image content.
  * @param contentScale [ContentScale] used to draw image content.
  * @param alignment [Alignment] used to draw the image content.
+ * @param modifier [Modifier] used to draw the image content.
  * @param fallbackContent The content to display with a thumbnail is unable to be loaded.
  */
 @Composable
 fun ThumbnailImage(
     request: ImageLoadRequest,
-    storage: ThumbnailStorage,
-    modifier: Modifier,
     contentScale: ContentScale,
     alignment: Alignment,
+    modifier: Modifier = Modifier,
```

Comment:
Is this just some unrelated bookkeeping? Fine with me if so, just checking

Reply:
Yes, Modifiers are supposed to be optional, so I opted to include that minor change now